spec/services/user_invite_service_spec.rb

```diff
@@ -50,5 +50,60 @@
       expect(result).to have_role(Role::ORG_ADMIN, organization)
       expect(result).not_to have_role(Role::PARTNER, :any)
     end
+
+    it "should create the user without name with role" do
+      result = nil
+      expect {
+        result = described_class.invite(name: "",
+          email: "email2@email.com",
+          roles: [Role::ORG_USER, Role::ORG_ADMIN],
+          resource: organization)
+      }.to change { ActionMailer::Base.deliveries.count }.by(1)
+      expect(result.name).to eq("")
+      expect(result.email).to eq("email2@email.com")
+      expect(result).to have_role(Role::ORG_USER, organization)
+      expect(result).to have_role(Role::ORG_ADMIN, organization)
+    end
+  end
+end
+
+RSpec.describe UserInviteService, type: :service do
```

Comment:
I think there should only be one overall `describe` block in a spec; does it work if you put this inside the other one?

Reply:
Unified the tests in one describe block. Thanks for the feedback!